tests/bot/cogs/moderation/test_utils.py

```diff
@@ -176,20 +176,18 @@ def setUp(self):
         test_cases = [
             {
                 "args": (self.user, "Test title", "Example content"),
-                "expected_output": {
-                    "description": "Example content",
-                    "title": "Test title",
-                    "icon_url": Icons.user_verified
-                },
+                "expected_output": Embed(
+                    description="Example content",
+                    colour=PARDON_COLOR
+                ).set_author(name="Test title", icon_url=Icons.user_verified),
                 "send_result": True
             },
             {
                 "args": (self.user, "Test title 1", "Example content 1", Icons.user_update),
-                "expected_output": {
-                    "description": "Example content 1",
-                    "title": "Test title 1",
-                    "icon_url": Icons.user_update
-                },
+                "expected_output": Embed(
+                    description="Example content 1",
+                    colour=PARDON_COLOR
+                ).set_author(name="Test title 1", icon_url=Icons.user_update),
```

Comment:
These embeds really only differ by the icon. Change the test cases so that only the icon is provided. The test can directly create the rest of the embed within the subtest.